script/util/rule.js

```diff
@@ -12,7 +12,6 @@
  * @typedef Check
  * @property {string} input
  * @property {Array<string>} output
- * @property {string} setting
```

Comment:
This is unused, the `tests` key is used everywhere instead. I dropped it instead of using a clearer name.